spa/src/components/dashboard/Dashboard.test.tsx

```diff
@@ -7,13 +7,16 @@ import Dashboard from './Dashboard';
 import { CONTENT_SLUG } from 'routes';
 
 import useConnectStripeAccount from 'hooks/useConnectStripeAccount';
+import useConnectMailchimp from 'hooks/useConnectMailchimp';
 import useFeatureFlags from 'hooks/useFeatureFlags';
 import useUser from 'hooks/useUser';
 
+jest.mock('components/common/Modal/AudienceListModal');
```

Comment:
```suggestion
jest.mock('components/common/Modal/AudienceListModal/AudienceListModal');
```

Mocks need to match the exact path of the file you've created the `__mock__` for. This is mocking `components/common/Modal/AudienceListModal/index.ts`, which misses the mock you had created.

Reply:
Oh 🤦 ... this might have been the issue all along. Thanks. Didn't see that import diff until now 🙃